src/portable_python/cpython.py

```diff
@@ -30,6 +43,37 @@ class Cpython(PythonBuilder):
 
     xenv_CFLAGS_NODIST = "-Wno-unused-command-line-argument"
 
+    def build_information(self):
+        """Yields key/value pairs to store as build information"""
+        yield "cpython", {
+            "prefix": self.setup.prefix,
+            "source": self.url,
+            "static": runez.joined(self.modules.selected) or None,
+            "target": PPG.target,
+            "version": self.version,
+        }
+        yield "configure-args", runez.joined(runez.short(x) for x in self.c_configure_args())
+        compiled_by = os.environ.get("PP_ORIGIN") or PPG.config.get_value("compiled-by")
+        bc = self.setup.build_context
+        yield "compilation-info", {
+            "compiled-by": compiled_by or "https://pypi.org/project/portable-python/",
+            "date": datetime.datetime.now().strftime("%Y-%m-%d %H:%M"),
+            "host-platform": runez.SYS_INFO.platform_info,
+            "ldd-version": PythonInspector.tool_version("ldd"),
+            "portable-python-version": runez.get_version(__package__),
+            "special-context": bc.isolate_usr_local and bc,
+        }
+        additional = PPG.config.get_value("manifest", "additional-info")
+        if additional:
+            res = {}
+            for k, v in additional.items():
+                if isinstance(v, str) and v.startswith("$"):
+                    v = os.environ.get(v[1:])
```

Comment:
Can you be more explicit about what the behavior is if the env var is missing?  I don't imagine that serializing a None value into yaml makes much sense.

Reply:
`None` are systematically filtered out by relevant calls such as `runez.joined()` or `runez.flattened()` where there is a `keep_empty=False` default parameter.

Reply:
I didn't expect json_sanitize to even have a feature that would remove Nones.  Looks like it does and that's the default.
https://github.com/codrsquad/runez/blob/main/src/runez/serialize.py#L182

This is non-obvious, can you comment either here or near the yaml dump that you're relying on it?